buildroot-external/board/pc/ova/hassos-hook.sh

```diff
@@ -10,7 +10,7 @@ function hassos_pre_image() {
     cp "${BINARIES_DIR}/barebox.bin" "${BOOT_DATA}/EFI/BOOT/BOOTx64.EFI"
     cp "${BR2_EXTERNAL_HASSOS_PATH}/bootloader/barebox-state-efi.dtb" "${BOOT_DATA}/EFI/barebox/state.dtb"
 
-    echo "console=tty1" > "${BOOT_DATA}/cmdline.txt"
+    echo "console=ttyS0 console=tty1" > "${BOOT_DATA}/cmdline.txt"
```

Comment:
🤷 I don't know if that work as suspected on all kind of hypervisors/vm to be a default option. I mean people can edit the cmdline.txt if they need it. If it works over all systems, why not.

Reply:
The problem is, if you run on a headless system, its rather hard to change it (since you need to enable graphical access via VNC or similar etc. to only change that setting).

From my testing it doesn't seem to cause problems when serial port is not enabled (tested with KVM and Virtual Box, in the latter no serial port is the default it seems).